index.html

```diff
@@ -2,19 +2,30 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Anyxz</title>
-    <meta property="og:title" content="Ez Video Background Semantic Segmentation" />
-    <meta property="og:type" content="website" />
-    <!-- Update reqd -->
-    <meta property="og:image" content="#" />
-    <meta property="og:description" content="Ez Change/Modify segmented video background!" />
-    <!-- Update reqd -->
-    <meta property="og:url" content="#" />
-    <link rel="shortcut icon" href="./assets/favicon.png" type="image/x-icon" />
-    <link rel="stylesheet" href="styles.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Alef&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/5.0.0-alpha1/css/bootstrap.min.css" />
+  <!-- i have set the viewport  -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- i am changing the title to improve seo -->
```

Comment:
Please update all these "i am changing...." to "Changed the..."
Tip: Code reviews should be in third person view.

Reply:
ok noted.